bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/console/ThingConsoleCommandExtension.java

```diff
@@ -26,16 +29,19 @@
  *
  * @author Dennis Nobel - Initial contribution
  * @author Thomas Höfer - Added localization of thing status
+ * @author Stefan Triller - Added trigger channel command
  */
 public class ThingConsoleCommandExtension extends AbstractConsoleCommandExtension {
 
     private static final String SUBCMD_LIST = "list";
     private static final String SUBCMD_CLEAR = "clear";
     private static final String SUBCMD_REMOVE = "remove";
+    private static final String SUBCMD_TRIGGER_CHANNEL = "trigger-channel";
```

Comment:
as mentioned in https://github.com/eclipse/smarthome/pull/3268#discussion_r112394362, I would think that "trigger" would suffice here.